drone/build/build.go

```diff
@@ -16,5 +16,7 @@ var Command = cli.Command{
 		buildDeclineCmd,
 		buildPromoteCmd,
 		buildRollbackCmd,
+		buildRollbackCmd,
```

Comment:
You have duplicated `buildRollbackCmd`?

Reply:
Nice catch, thank you! I amended my commit to avoid pollution in the commit messages.